packages/flutter/lib/src/rendering/table.dart

```diff
@@ -517,6 +517,8 @@ class RenderTable extends RenderBox {
   /// Row decorations fill the horizontal and vertical extent of each row in
   /// the table, unlike decorations for individual cells, which might not fill
   /// either.
+  ///
+  /// _rowDecorations and _rowDecorationPainters need to be in sync.
```

Comment:
Private members don't really belong in a public doc comment. What does the user of RenderTable need to do to ensure that?

Reply:
Maybe this should just be a comment starting with `//` instead of `///`? And maybe move it one line down?